client/src/components/ReportTable.js

```diff
@@ -143,7 +143,7 @@ const ReportTable = ({
         <Table striped>
           <thead>
             <tr>
-              {showAuthors && <th>Author</th>}
+              {showAuthors && <th>Authors</th>}
```

Comment:
`showAuthors` seems to be redundant as `ReportTable` is only used in `ReportCollection` and `showAuthors` is always true there.